chpl/chpl-service/src/main/java/gov/healthit/chpl/util/Util.java

```diff
@@ -195,4 +142,191 @@ public static String formatCriteriaNumber(CertificationCriterionDTO criterion) {
         }
         return result;
     }
+
+    private static List<String> certificationResultSortOrder() {
```

Comment:
This isn't my favorite... now if we were to add a new criteria we have to mess with the lookup.properties file (which is basically a copy of our lookup tables in the database...) and have to also remember to change this list. I'm not sure what change to suggest but if I think of something I will post it.